JamrulesRPM

```diff
@@ -0,0 +1,313 @@
+# You will probably need to edit this file to get everything to compile.
```

Comment:
It looks like the only difference in this `Jamrules` file is that `USESDL` is disabled.  It'd be cleaner to use the original `Jamrules`, disabling `USESDL` on the command line, i.e.:

```
jam -sUSESDL=no
```
during the build step.  That way this file won't have to be kept in sync with the original.

Reply:
Thanks for this, very helpful, will make the change!

Reply:
I attempted a build using the commandline approach you mention but the build system does not seem to honor the commandline flag.  Still getting SDL errors in the build.  Any thoughts?

Reply:
Checked in changes.